src/main/java/ro/unibuc/triplea/domain/auth/model/entity/meta/User.java

```diff
@@ -41,10 +41,9 @@ public class User implements UserDetails {
     @Enumerated(EnumType.STRING)
     private Gender gender;
 
-    @Column(name = "Email", nullable = false, length = 100, unique = true)
-    private String email;
     private String password;
 
+    @Column(name = "Email", nullable = false, length = 100, unique = true)
     private String username;
```

Comment:
Column with name email but field named username - we should look into this inconsistency

Reply:
We can leave it like this after our discussion: the username field is from the UserDetails model of Spring Security, but in our business logic it's an email